cmd/booster-http/run.go

```diff
@@ -262,32 +254,25 @@ var runCmd = &cli.Command{
 	},
 }
 
-func parseSupportedResponseFormats(cctx *cli.Context) []string {
+func parseSupportedResponseFormats(cctx *cli.Context) ([]string) {
```

Comment:
I don't think we need parseSupportedResponseFormats anymore. We can probably replace the call to it with code that checks that serve-gateway is set to verifiable / all / none

Reply:
we can change this method to return a bool. If it's none, then it'll return false.

```
func shouldEnableIPFSGateway(cctx *cli.Context) (bool) {
	switch cctx.String("serve-gateway") {
	case "verifiable":
	case "all":
		return true
	}
	return false
}
```

Reply:
added the new function and replace the code with it.